### ISSUE TITLE
Ruby App: code converter not found Error 

### ISSUE THREAD

**Issue by sshaw** [nginx/unit#92]
I noticed this when trying to run an app that uses [the SASS gem](https://rubygems.org/gems/sass). 

Here's a simple example:

```
# config.ru
x = "a".encode("UTF-16BE").force_encoding("BINARY")
run ->(env) do
  [200, {"Content-Type" => "text/plain"}, ["Oizinho!"]]
end
```

It results in the following: 

```
2018/04/02 00:33:59 [info] 58382#19100772 "simple" application started
2018/04/02 00:33:59 [alert] 58382#19100772 Ruby: Failed to parse rack script
2018/04/02 00:33:59 [alert] 58382#19100772 Ruby: /Users/sshaw/code/ruby/rack-stuff/nginx-unit-test/config.ru:3:in `encode': code converter not found (UTF-8 to UTF-16BE) (Encoding::ConverterNotFoundError)
2018/04/02 00:33:59 [alert] 58382#19100772 from /Users/sshaw/code/ruby/rack-stuff/nginx-unit-test/config.ru:3:in `block in <main>'
2018/04/02 00:33:59 [alert] 58382#19100772 from /Users/sshaw/.rvm/gems/ruby-2.3.5/gems/rack-2.0.4/lib/rack/builder.rb:55:in `instance_eval'
2018/04/02 00:33:59 [alert] 58382#19100772 from /Users/sshaw/.rvm/gems/ruby-2.3.5/gems/rack-2.0.4/lib/rack/builder.rb:55:in `initialize'
2018/04/02 00:33:59 [alert] 58382#19100772 from /Users/sshaw/code/ruby/rack-stuff/nginx-unit-test/config.ru:in `new'
2018/04/02 00:33:59 [alert] 58382#19100772 from /Users/sshaw/code/ruby/rack-stuff/nginx-unit-test/config.ru:in `<main>'
2018/04/02 00:33:59 [alert] 58382#19100772 from /Users/sshaw/.rvm/gems/ruby-2.3.5/gems/rack-2.0.4/lib/rack/builder.rb:49:in `eval'
2018/04/02 00:33:59 [alert] 58382#19100772 from /Users/sshaw/.rvm/gems/ruby-2.3.5/gems/rack-2.0.4/lib/rack/builder.rb:49:in `new_from_string'
2018/04/02 00:33:59 [alert] 58382#19100772 from /Users/sshaw/.rvm/gems/ruby-2.3.5/gems/rack-2.0.4/lib/rack/builder.rb:40:in `parse_file'
```

This does not occur when running via `rackup`.

**Comment by lexborisov** [comment]
Hi, @sshaw !
Please, try this patch:

```patch
# HG changeset patch
# User Alexander Borisov <alexander.borisov@nginx.com>
# Date 1522761982 -10800
#      Tue Apr 03 16:26:22 2018 +0300
# Node ID 434041aed57b9faa4e0523b0e876222554b884cc
# Parent  ced9f2afd5a5963f57b8bbc34c12521f0893a580
Ruby: added 'enc/trans/transdb' module for converting encodings.

This closes #92 issue on GitHub.

diff --git a/src/ruby/nxt_ruby.c b/src/ruby/nxt_ruby.c
--- a/src/ruby/nxt_ruby.c
+++ b/src/ruby/nxt_ruby.c
@@ -148,6 +148,9 @@ nxt_ruby_init_basic(VALUE arg)
         return Qnil;
     }
 
+    rb_funcall(rb_cObject, rb_intern("require"), 1,
+               rb_str_new2("enc/trans/transdb"));
+
     rc = nxt_ruby_init_io(rack_init->task);
     if (nxt_slow_path(rc != NXT_OK)) {
         return Qnil;

```
Our tests showed that the problem was gone. If there are no questions then I will add this patch to master.

Thanks!

**Comment by sshaw** [comment]
Worked for me, thanks!
